### PR TITLE
Lazy check

### PR DESCRIPTION
**Pull request by calvinmetcalf**
fix for #16 no longer does define property inside a try catch because anything that doesn't have that isn't going to have a working CRNG

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
built off of #18

File: test.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ while (++i < 2) {
 	run(i);
 }
 function isNode10() {
-  return process.version && process.version.split('.').length === 3 && parseInt(process.version.split('.')[1], 10) <= 10;
+  return process.version && process.version.split('.').length === 3 && parseInt(process.version.split('.')[1], 10) === 10 && parseInt(process.version.split('.')[0] === 0)
```
Comment (dcousens) [inline test.js]: maybe just split it once

File: test.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ var mods = [
    'modp1', 'modp2', 'modp5', 'modp14', 'modp15', 'modp16', 'modp17', 'modp18'
 ];
 function isNode10() {
-  return process.version && process.version.split('.').length === 3 && parseInt(process.version.split('.')[1], 10) <= 10;
+  return process.version && process.version.split('.').length === 3 && parseInt(process.version.split('.')[1], 10) === 10 && process.version.split('.')[0] === 'v0';
 }
```
Comment (dcousens) [inline test.js]: is this a double up?

**Comment by dcousens** [comment]
LGTM, just need to address the nits

**Comment by calvinmetcalf** [comment]
not actually green don't know why it says it is

**Comment by calvinmetcalf** [comment]
and we are actually green

File: lib/dh.js [hunk 3]
```diff
@@ -100,16 +89,23 @@ function DH(prime, generator, malleable) {
   this._primeLen = prime.length;
   this._pub = void 0;
   this._priv = void 0;
-  
+  this._primeCode = void 0;
```
Comment (dcousens) [inline lib/dh.js]: This is really uncommon, could you put a comment to explain what the `void 0` achieves here?

Comment (calvinmetcalf) [inline lib/dh.js]: We can also just change then to undefined

Comment (dcousens) [inline lib/dh.js]: Please do then.

Comment (calvinmetcalf) [inline lib/dh.js]: done

**Comment by dcousens** [comment]
:+1:

**Pull request merged**